### PR TITLE
Fixed #27701 - Document staticfiles runserver bypassing of middleware

### PR DESCRIPTION
**Pull request by Jkrzy**

### CODE REVIEW

File: docs/ref/contrib/staticfiles.txt [hunk 1]
```diff
@@ -230,6 +230,11 @@ Example usage::
 
 .. _staticfiles-storages:
```
Comment (atombrella) [inline docs/ref/contrib/staticfiles.txt]: Please put your changes above this line, so internal doc links are still working.

File: docs/ref/contrib/staticfiles.txt [hunk 2]
```diff
@@ -230,6 +230,11 @@ Example usage::
...
 
+.. note::
+
+  When serving staticfiles, runserver delivers the files directly and bypasses
```
Comment (felixxm) [inline docs/ref/contrib/staticfiles.txt]: `staticfiles` -> `static files`
`runserver` ->``:djadmin:`runserver` ``
`middleware` -> `middlewares`

File: docs/ref/contrib/staticfiles.txt [hunk 3]
```diff
@@ -228,6 +228,11 @@ Example usage::
 
     django-admin runserver --insecure
 
+.. note::
+
+  When serving static files, :djadmin:`runserver` delivers the files directly
+  and bypasses all installed middlewares.
```
Comment (timmartin) [inline docs/ref/contrib/staticfiles.txt]: Sorry to be pedantic, but I'm not sure about using the plural "middlewares". The documentation isn't entirely consistent ("middlewares" does have a couple of uses), but the most common approach seems to be to treat "middleware" as a mass noun, i.e. not modifying it for number. For example, in `docs/ref/applications.txt`

    Applications include some combination of models, views, templates, template
    tags, static files, URLs, middleware, etc.

or in `docs/ref/request-response.txt`:

    Some of the middleware included in Django's contrib apps set attributes on the
    request.

Comment (Jkrzy) [reply docs/ref/contrib/staticfiles.txt]: Agreed, and this plural issue has been mentioned before. 
https://code.djangoproject.com/ticket/27376#comment:3

I'll submit a separate issue to address the consistency of it across the docs.

**Pull request merged**